src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/AggregateRelationCollectionSource.cs

```diff
@@ -73,8 +75,52 @@ public void SetCollection(IAggregateRelationCollection collection)
                 PropertyChanged?.Invoke(this, KnownEventArgs.HasItemsPropertyChanged);
             };
 
+            // Work around inconsistent use of IPrioritizedComparable in the tree view control
+            var view = (ListCollectionView)CollectionViewSource.GetDefaultView(_collection);
+            view.CustomSort = PrioritizedComparableComparer.Instance;
```

Comment:
This feels like an implementation detail, can we not fix the source? Or can we file a bug to fix this?

Reply:
Agree. Filed https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1158136.